src/main/java/net/syscon/elite/api/model/Alias.java

```diff
@@ -1,203 +1,57 @@
 package net.syscon.elite.api.model;
 
-import com.fasterxml.jackson.annotation.*;
+import com.fasterxml.jackson.annotation.JsonInclude;
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.EqualsAndHashCode;
-import lombok.NoArgsConstructor;
+import lombok.*;
```

Comment:
think people have started to use the non wildcards - if everyone standardises on that then it leads to less churn

Reply:
IntelliJ has become such a pain with that, it autowildcards now after a certain amount, so you have to be vigilant :D I am still on the hunt for a setting to stop it from doing that.

Reply:
It's annoying that it doesn't have a unlimited - but you can set it to 999 and its per package I think.